v1/openapi/requests/jet-drops-byjetid.yaml

```diff
@@ -1,6 +1,6 @@
 get:
   summary: Jet drops by jet ID
-  description: Gets jet drops by `jet_id` and based on filtering and pagination parameters.
+  description: Gets jet drops by `jet_id` and based on specified filtering and pagination parameters.
```

Comment:
How do I "base" a getter on filtering and pagination parameters?
How about that:
Gets an array of jet drops given a `jet_id` as a path (or query?) parameter.
To invoke this request, specify filtering and pagination parameters. (and you can elaborate on them if you feel that it's not obvious how to use all the parameters) (btw, are the parameters optional?)